src/redux/RoomSlice.js

```diff
@@ -0,0 +1,11 @@
+// import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+async function fetchRoomDetail(id) {
+  const apiUrl = `http://localhost:3000/api/v1/rooms/${id}`;
+  const response = await axios.get(apiUrl);
+  const responseData = await response.json();
+  return responseData;
+}
+
+export default fetchRoomDetail;
```

Comment:
- The function `fetchRoomDetails()` is created and `exported` but never `imported`. Make sure that you remove this part from your project or interact with a configured Redux reducer and then import it in the `src/Pages/roomDetail.js` file